eng/scripts/Language-Settings.ps1

```diff
@@ -118,6 +118,28 @@ function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
   return $allPackageProps
 }
 
+function Get-rust-AdditionalValidationPackagesFromPackageSet ($packagesWithChanges, $diff, $allPackageProperties) {
```

Comment:
Nit: Odd not to uppercase Rust when everything else is, and PowerShell recommends `Verb-CompoundNoun` with only a single hyphen, but I know EngSys violates that quite a bit already.
```suggestion
function Get-RustAdditionalValidationPackagesFromPackageSet ($packagesWithChanges, $diff, $allPackageProperties) {
```

Reply:
this is the pattern for all the languages.  I agree it's odd, and I assume that something broke somewhere when someone tried to caps it:

```ps
Get-dotnet-AdditionalValidationPackagesFromPackageSet
Get-go-PackageInfoFromPackageFile
Get-python-DocsMsMetadataForPackage
```

Reply:
The names are templated in `eng/common/scripts/common.ps1`:
https://github.com/Azure/azure-sdk-for-python/blob/main/eng/common/scripts/common.ps1#L45
```ps
$GetPackageInfoFromRepoFn = "Get-${Language}-PackageInfoFromRepo"
$GetPackageInfoFromPackageFileFn = "Get-${Language}-PackageInfoFromPackageFile"
$PublishGithubIODocsFn = "Publish-${Language}-GithubIODocs"
$UpdateDocsMsPackagesFn = "Update-${Language}-DocsMsPackages"
$GetDocsMsMetadataForPackageFn = "Get-${Language}-DocsMsMetadataForPackage"
$GetDocsMsDevLanguageSpecificPackageInfoFn = "Get-${Language}-DocsMsDevLanguageSpecificPackageInfo"
$GetGithubIoDocIndexFn = "Get-${Language}-GithubIoDocIndex"
```

Where $Language comes from LanguageSettings.ps1 and is lowercase in all repos.